must-gather/collection-scripts/gather_nodes

```diff
@@ -79,3 +81,6 @@ wait "${ADM_PIDS[@]}"
 oc delete -f $DAEMONSET_MANIFEST
 oc delete -f $SERVICEACCOUNT_MANIFEST
 oc delete -f $NAMESPACE_MANIFEST
+
+# force disk flush to ensure that all data gathered is accessible in the copy container
+sync
```

Comment:
Is this really needed? I though sync just makes sure the underlying storage has it all. Applications go through kernel and get the storage+caches version.. So I am curious

Reply:
Not sure, it's done pretty frequently in the (OCP) must-gather scripts though. Can be some cargo cult or a workaround of an obscure bug.

Reply:
that's the only thing I was wondering about too.  everything else looks fine

Reply:
ok, I'll remove the `sync`s until proven needed (with a "why" attached)